pom.xml

```diff
@@ -638,7 +638,7 @@
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro-mapred</artifactId>
-      <classifier>hadoop2</classifier>
+<!--      <classifier>hadoop2</classifier>-->
```

Comment:
Why is it commented ?

Reply:
due to the version issue it is commented.